nsv13/code/modules/overmap/types/nanotrasen.dm

```diff
@@ -328,3 +328,47 @@
 /obj/structure/overmap/nanotrasen/ai/fighter/apply_weapons()
 	weapon_types[FIRE_MODE_ANTI_AIR] = new/datum/ship_weapon/light_cannon(src)
 	weapon_types[FIRE_MODE_MISSILE] = new/datum/ship_weapon/missile_launcher(src)
+
+//Dominion
```

Comment:
Sort these into the relevant sections;
- base
- player version
- ai version

rather than just dumping in the bottom of the file

Reply:
Sorted.